src/preamble.js

```diff
@@ -2223,6 +2223,7 @@ function integrateWasmJS(Module) {
       exports = instance.exports;
       if (exports.memory) mergeMemory(exports.memory);
       Module['asm'] = exports;
+      asm = exports;
```

Comment:
+1, in general I think we should try to avoid having our code use the global variables as much as possible.